GrandCentralBoardTests/GitHubDataProviderTests.swift

```diff
@@ -0,0 +1,93 @@
+//
+//  GitHubDataProviderTests.swift
+//  GrandCentralBoard
+//
+//  Created by Michał Laskowski on 27.04.2016.
+//  Copyright © 2016 Oktawian Chojnacki. All rights reserved.
+//
+
+import XCTest
+import Nimble
+import RxSwift
+import OHHTTPStubs
+@testable import GrandCentralBoard
+
+
+final class GitHubDataProviderTests: XCTestCase {
+
+    private var urlRequest: NSURLRequest!
+
+    private let jsonReposArray: [[String: AnyObject]] = {
+        let path = NSBundle(forClass: GitHubDataProviderTests.self).pathForResource("repos", ofType: "json")!
+        let jsonData = NSData(contentsOfFile: path)!
+        return try! NSJSONSerialization.JSONObjectWithData(jsonData, options: []) as! [[String: AnyObject]]
+    }()
+    private let pullRequestsArray: [AnyObject] = {
+        let path = NSBundle(forClass: GitHubDataProviderTests.self).pathForResource("pull_requests", ofType: "json")!
+        let jsonData = NSData(contentsOfFile: path)!
+        return try! NSJSONSerialization.JSONObjectWithData(jsonData, options: []) as! [AnyObject]
+    }()
+
+
+    override func setUp() {
+        urlRequest = nil
+
+        stub(isPath("/user/repos")) { urlRequest in
+            self.urlRequest = urlRequest
+            return OHHTTPStubsResponse(JSONObject: self.jsonReposArray, statusCode: 200, headers: ["Content-Type":"application/json"])
+        }
+
+        stub(pathStartsWith("/repos/")) { _ in
+            return OHHTTPStubsResponse(JSONObject: self.pullRequestsArray, statusCode: 200, headers: ["Content-Type":"application/json"])
+        }
+    }
+
+    override func tearDown() {
+        OHHTTPStubs.removeAllStubs()
+    }
+
+
+    func testAreHeadersValid() {
+        let token = "token"
```

Comment:
I'd change `"token"` to something like `"123"` or `"sample-value"` because later on it's unclear why we have `token` string twice in : `Authorization: token token`.